app/serializers/api/v1/offer_summary_serializer.rb

```diff
@@ -83,5 +99,18 @@ def goodcity_user?
       User.current_user.present?
     end
 
+    def restrict_payload_for_offers_package?
+      !options[:restrict_payload_for_offers_package]
+    end
+
+    %w[include_state? include_created_at? include_inactive_at?
+      include_updated_at? include_submitted_at? include_reviewed_at? include_review_completed_at?
+      include_received_at? include_cancelled_at? include_start_receiving_at?
+      include_submitted_items_count? include_accepted_items_count? include_rejected_items_count?
+      include_expecting_packages_count? include_missing_packages_count? include_received_packages_count?
+      include_display_image_cloudinary_id? include_notes? include_closed_by? include_reviewed_by?
+      include_received_by? include_delivery?].each do |method|
+        alias_method method.to_sym, :restrict_payload_for_offers_package?
+      end
```

Comment:
I'm assuming you're trying to exclude those that you do not need, but those seem to be almost ALL of them.
I think at this point we could consider not using a serializer at all